PVTPackage/source/Utils/math.hpp

```diff
@@ -14,187 +14,220 @@
+  return sum_array( temp );
 }
 
-template <typename Array>
-typename Array::value_type prod_array(const Array& args)
+template< typename Array >
+typename Array::value_type prod_array( const Array & args )
 {
-  return std::accumulate(args.begin(), args.end(), Array::value_type(1), std::multiplies<typename Array::value_type>());
+  return std::accumulate( args.begin(), args.end(), Array::value_type( 1 ), std::multiplies< typename Array::value_type >() );
 }
 
-template <typename... Ts, typename std::common_type<int, Ts...>::type* = nullptr>
-typename std::common_type<int, Ts...>::type prod(Ts... args)
+template< typename... Ts, typename std::common_type< int, Ts... >::type * = nullptr >
+typename std::common_type< int, Ts... >::type prod( Ts... args )
 {
   // With C++17 fold expressions we could just write:
   // return (0 * ... * args);
-  using type = typename std::common_type<int, Ts...>::type;
-  std::array<type, sizeof...(args)> temp{ args... };
-  return prod_array(temp);
+  using type = typename std::common_type< int, Ts... >::type;
+  std::array< type, sizeof...( args ) > temp{ args... };
+  return prod_array( temp );
 }
 
-template<typename T>
-bool isNonNegative(const T& in)
+template< typename T >
+bool isNonNegative( const T & in )
 {
   return in >= 0;
 }
 
-template<typename T>
-bool isNonNegative(const std::vector<T>& in)
+template< typename T >
+bool isNonNegative( const std::vector< T > & in )
 {
   bool res = true;
-  for (std::size_t i = 0; i != in.size(); ++i)
-    if (in[i] < std::numeric_limits<T>::epsilon()) return false;
+  for( std::size_t i = 0; i != in.size(); ++i )
+  {
+    if( in[i] < std::numeric_limits< T >::epsilon() )
+    { return false; }
+  }
   return res;
 }
 
-template<typename T>
-bool isLowerThanOne(const std::vector<T>& in)
+template< typename T >
+bool isLowerThanOne( const std::vector< T > & in )
 {
   bool res = true;
-  for (std::size_t i = 0; i != in.size(); ++i)
-    if (in[i] > 1 ) return false;
+  for( std::size_t i = 0; i != in.size(); ++i )
+  {
+    if( in[i] > 1 )
+    { return false; }
+  }
   return res;
 }
 
-template<typename T>
-bool isPositive(const T& in)
+template< typename T >
+bool isPositive( const T & in )
 {
-  return in > std::numeric_limits<T>::epsilon();
+  return in > std::numeric_limits< T >::epsilon();
 }
 
-template<typename T>
-bool isPositive(const std::vector<T>& in)
+template< typename T >
+bool isPositive( const std::vector< T > & in )
 {
   bool res = true;
-  for (std::size_t i = 0; i != in.size(); i++)
-    if (in[i] + std::numeric_limits<T>::epsilon() <= 0) return false;
+  for( std::size_t i = 0; i != in.size(); i++ )
+  {
+    if( in[i] + std::numeric_limits< T >::epsilon() <= 0 )
+    { return false; }
+  }
   return res;
 }
 
-template<typename T>
-bool isNearlyEqual(T in, T val, int ulp = 1) {
+template< typename T >
+bool isNearlyEqual( T in,
+                    T val,
+                    int ulp = 1 )
```

Comment:
@TotoGaz I did not go through any of the files in the `Utils` folder. I know that  since`PVTPackage` is a submodule of GEOSX, it is difficult to use some code from the main repository. But, functionalities like math functions, interpolation from a table, file reading, logging, etc already exist in GEOSX and maybe that can be a good argument to make `PVTPackage` a part of the main repo as some point to avoid duplication.

Reply:
> @TotoGaz I did not go through any of the files in the `Utils` folder.

I did remove all the unused.
And reindent the stuff to be consistent.
That's it.

> I know that since`PVTPackage` is a submodule of GEOSX, it is difficult to use some code from the main repository. But, functionalities like math functions, interpolation from a table, file reading, logging, etc already exist in GEOSX and maybe that can be a good argument to make `PVTPackage` a part of the main repo as some point to avoid duplication.

There may surely be interesting features to grab from GEOSX's utility modules, yes.